scripts/Packages/Package_DFN_QFN/size_definitions/qfn-24.yaml

```diff
@@ -401,6 +401,62 @@ QFN-24-1EP_4x4mm_P0.5mm_EP2.7x2.7mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+QFN-24-1EP_4x4mm_P0.5mm_EP2.8x2.8mm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.analog.com/media/en/technical-documentation/data-sheets/hmc431.pdf'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    nominal: 4
+    tolerance: 0.1
+  body_size_y:
+    nominal: 4
+    tolerance: 0.1
+  body_height:
+    nominal: 0.9
+    tolerance: 0.1
+
+  lead_width:
+    nominal: 0.24
+    tolerance: 0.06
+  lead_len:
+    nominal: 0.3
+    tolerance: 0.1
+
+  EP_size_x:
+    nominal: 2.8
+    tolerance: 0.15
+  EP_size_y:
+    nominal: 2.8
+    tolerance: 0.15
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [2, 2]
+
+  thermal_vias:
+    count: [3, 3]
+    drill: 0.2
+    # min_annular_ring: 0.15
+    paste_via_clearance: 0.1
+    # EP_num_paste_pads: [2, 2]
+    #paste_between_vias: 1
+    #paste_rings_outside: 1
+    EP_paste_coverage: 0.75
+    #grid: [1, 1]
+    # bottom_pad_size:
+    paste_avoid_via: False
+
+  pitch: 0.5
```

Comment:
please add a comment that this package strangely does not give the pitch as a reference dimension but with a tolerance range. (In case IPC ever adds a rule how this is to be handled.)